### DISCUSSION TITLE
Xabier Gamboa

### DISCUSSION THREAD

**Discussion by xglantz** [community/community#129393] [New to GitHub]
Quiero subir un proyecto al repositorio pero no puedo pegar carpetas de archivos. Necesito algo más que la versión gratuita?

**Comment by davevad93** [comment]
Hi @xglantz ,

Are the folders you are trying to upload larger than 2GB? If so the problem is that `GitHub` has a file size limit, you should consider using `Git LFS`, take a look at this from the docs:

- [About large files on GitHub](https://docs.github.com/es/repositories/working-with-files/managing-large-files/about-large-files-on-github) 
- [About Git Large File Storage](https://docs.github.com/es/repositories/working-with-files/managing-large-files/about-git-large-file-storage)
- [Installing Git Large File Storage](https://docs.github.com/es/repositories/working-with-files/managing-large-files/installing-git-large-file-storage)

**Comment by grandmas-favorite** [comment]
Hi @xglantz! Thanks for participating!

We take our[Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
